packages/wasmkit/src/lib/deploy/contract.ts

```diff
@@ -45,6 +45,25 @@ export class Contract {
   private checkpointData: Checkpoints;
   private readonly checkpointPath: string;
 
+  private loadInterval?: NodeJS.Timer;
+
+  private printLoadingAnimation (logMsg: string): void {
+    const loadingChars = ['/', '-', '\\', '|'];
+    let index = 0;
+
+    this.loadInterval = setInterval(() => {
+      process.stdout.write(`${logMsg} ${loadingChars[index]}\r`);
```

Comment:
Why can't we use console.log() here?

Also, logMsg shoudl be after loading char